resources/features/Stratification.feature

```diff
@@ -0,0 +1,69 @@
+            | 1100000004 | 9906000145 | AUBASOT(CHRISTCHURCH) LIMITED |              | 1 GARTH EDGE             | SHAWFORTH      | WHITWORTH   | ROCHDALE LANCASHIRE     |          | OL12 8EH | 2           | 45320 | 0           | 0         | 7            | 7            | 0            | 0             | 0            | 0.509298879 |
+            | 1100000005 | 9906000175 | HIBAER                        |              | 1 GEORGE SQUARE          | GLASGOW        |             |                         |          | G2 5LL   | 1           | 45400 | 1           | 1         | 106          | 106          | 0            | 0             | 0            | 0.147768898 |
+            | 1100000006 | 9906000205 | HIBAER                        |              | 1 GLEN ROAD              | HINDHEAD       | SURREY      |                         |          | GU26 6QE | 1           | 46110 | 1           | 1         | 297          | 297          | 0            | 0             | 0            | 0.588701588 |
+            | 1100000007 | 9906000275 | IBANOCTRACTS UK LTD           |              | 1 GLYNDE PLACE           | HORSHAM        | WEST SUSSEX |                         |          | RH12 1NZ | 1           | 46120 | 2           | 2         | 287          | 287          | 0            | 0             | 0            | 0.155647458 |
+            | 1100000008 | 9906000325 | TLUBARE                       |              | 1 GORSE ROAD             | REYDON         | SOUTHWOLD   |                         |          | IP18 6NQ | 1           | 45130 | 13          | 3         | 197          | 197          | 0            | 0             | 0            | 0.446872271 |
+            | 1100000009 | 9906000355 | BUCARR                        |              | 1 GRANVILLE AVENUE       | LONG EATON     | NOTTINGHAM  |                         |          | NG10 4HA | 1           | 46140 | 1           | 1         | 18           | 18           | 0            | 0             | 0            | 0.847311602 |
+            | 1100000010 | 9906000405 | DCAJ&WALTON                   |              | 1 GRANVILLE AVENUE       | LONG EATON     | NOTTINGHAM  |                         |          | NG10 4HA | 1           | 46150 | 2           | 2         | 72           | 72           | 0            | 0             | 0            | 0.548604086 |
+            | 1100000011 | 9906000415 | &BAMCFLINT                    |              | 1 GARENDON WAY           | GROBY          | LEICESTER   |                         |          | LE6 0YR  | 1           | 46160 | 1           | 0         | 400          | 400          | 0            | 0             | 0            | 0.269071541 |
+        And a Strata of selection type Universal from Stratification Properties file:
```

Comment:
Selection type is irrelevant to the stratification process and may lead to confusion if you reference it, it is only used for sampling. The test shouldn't be for selection type Universal otherwise you need to create test for each type - the response for all three should be the full allocation of units to the cell_no.

Reply:
Yes, you're right. I did this as it would link to the And("""a Strata of selection type [^\s]+ from Stratification Properties file:$""") but I will change it to And("""a Strata of [^\s]* from Stratification Properties file:$""")